scripts/ci-checks.sh

```diff
@@ -33,7 +33,7 @@ function endgroup() {
 
 
 group "Shell scripts"
-git ls-files | grep -e '\.sh$' | grep -E -v "^3rdparty" | xargs shellcheck -s bash -e SC2044,SC2002,SC1091,SC2181
+git ls-files | grep -e '\.sh$' | grep -E -v "^3rdparty" | xargs shellcheck -s bash -e SC2044,SC2002,SC1091,SC2181,SC2230
```

Comment:
Went to look for SC2230, which is usage of `which`, flagged for reasons that are entirely irrelevant to us. It got me wondering what default level we use, and it's "style", which is why we pick up these less than helpful issues. Proposed fix: https://github.com/microsoft/CCF/pull/6784